example/ios/Podfile.lock

```diff
@@ -10,8 +10,8 @@ PODS:
     - React-Core (= 0.66.1)
     - React-jsi (= 0.66.1)
     - ReactCommon/turbomodule/core (= 0.66.1)
-  - Fidel (1.10.1)
```

Comment:
Why is this Fidel one `1.10.3` and not `1.10.2`?

Reply:
This is the version used by the iOS example project. It was never updated to `1.10.2`.